### DISCUSSION TITLE
Topics winget

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-AutoUpdate#233] [Show and tell]
Now, a tie (7:th place):
https://github.com/topics/winget
![image](https://user-images.githubusercontent.com/102996177/200692642-ab657547-17da-43bd-b9af-50164dd355d8.png)

**Comment by Romanitho** [comment]
even better :p

**Reply by KnifMelti** [reply]
On the way up (7:th in the moment):

![image](https://user-images.githubusercontent.com/102996177/205343199-6949fc73-c727-4b18-a884-ae24d1ed9dad.png)

**Reply by Romanitho** [reply]
🎉

**Comment by KnifMelti** [comment]
And, there it is, now 6:th place and leaving [HandyWinGet](https://github.com/HandyOrg/HandyWinGet) behind 😉

![image](https://user-images.githubusercontent.com/102996177/210625195-1569a0b7-4198-416a-99f2-84708748363b.png)

**Reply by Romanitho** [reply]
haha :)

**Comment by Romanitho** [comment]
Never been that close to 500 stars 😅

**Reply by Romanitho** [reply]
and voilà !!! 500+ 🍾🍾
Very happy for this project. Thanks to all contributors and users for making this thing grow.

**Reply by KnifMelti** [reply]
Then (maybe) you could:
"I work on open source projects, can I get access to GitHub Copilot for free?
**_People who maintain popular open source projects receive a credit to have 12 months of GitHub Copilot access for free_**. A maintainer of a popular open source project is defined as someone who has write or admin access to one or more of the most popular open source projects on GitHub. **_Simply visit the GitHub Copilot subscription page to see if you are one of the open source maintainers that meet our criteria for a complimentary subscription. If you do, you should see that you can add GitHub Copilot for no charge_**. If you see a charge on the purchase page then this means that you do not qualify at this time. Once awarded, **_if you are still a maintainer of a popular open source project when your initial 12 months subscription expires then you will be able to renew your subscription for free_**."

**Reply by Romanitho** [reply]
apparently, not eligible yet :/

**Comment by KnifMelti** [comment]
Sneaked up and passed **Windows 11 fixer** now.!
4:th place
https://github.com/topics/winget

**Reply by Romanitho** [reply]
🤯
